### PR TITLE
Update utils.py

### PR DESCRIPTION
**Pull request by parthapratimkalita**
Solved the divided by zero problem.

# Description

Provide a high-level summary of the change and which issue it relates to. 
Include relevant motivation and context.

*If the issue is on Github, simply link to it using the '#' symbol)*:
- resolves #(issue)

# Type of change

*Delete as appropriate:*
- **fix**: A bug fix
- **feat**: A new feature
- **refactor**: A code change that neither fixes a bug nor adds a feature
- **docs**: Documentation only changes
- **test**: Adding missing tests or correcting existing tests
- **perf**: A code change that improves performance
- **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
- **ci**: Changes to our CI configuration files and scripts (i.e., those in the .github directory)
- **build**: Changes that affect the build system or external dependencies (i.e., poetry, conda)

If this is a breaking change (which might apply to any of the other kinds of change), note that below, otherwise delete 
this section:
- **BREAKING_CHANGE**: {description}

> Update the title of the PR to follow the convention:
> `{type}{! if BREAKING_CHANGE}: {short description}`, e.g. 
> - "fix: add missing foo in module bar" 
> - "feat: new theorist abc",
> - "feat!: replace module x with module y"
> - "docs: add notebooks on theorist z" 
> 
> ... then delete this block.

# Features (Optional)
- #3 

# Questions (Optional)
- potential questions for the code reviewer (e.g., if you want feedback on a certain method or class; any feedback required from other code contributors with their mentioning)

# Remarks (Optional)
- any additional remarks for the code reviewer
- if your remarks or questions pertain to individual lines in the code, you may also include them as comments in the code review window

# Screenshots (Optional)
If your PR is related to UI change or anything visual, feel free to include screenshots to give the reviewer straightforward understandings on your changes.

### CODE REVIEW

File: src/utils.py [hunk 1]
```diff
@@ -103,4 +107,4 @@ def return_random_number() -> int:
     float
     '''
 
-    return np.random.randint(0, 100)
\ No newline at end of file
+    return np.random.randint(0, 100)
```
Comment (Krishnendu-Bose) [inline src/utils.py:110]: The new line is an exact copy of the previous line. So, this change doesn't make any sense.

**Review by AlexanderDitz — COMMENTED** [review]
nice work :)

File: src/utils.py [hunk 2]
```diff
@@ -41,7 +41,11 @@ def divide(a: float, b: float) -> float:
     Returns:
     float
     '''
-    return a / b
+    try:
+        return a / b
+    except ZeroDivisionError:
+        print("Error: Division by zero is not allowed.")
+        return None
```
Comment (Krishnendu-Bose) [inline src/utils.py:48]: The revision makes sense. We can merge this with the main code.

**Review by Krishnendu-Bose — CHANGES_REQUESTED** [review]
Please add the necessary modifications according to my comments and then we can merge the your commit to the main.

**Comment by parthapratimkalita** [comment]
yes

**Pull request closed without merging**